.github/workflows/customer-service.yml

```diff
@@ -18,10 +22,27 @@ jobs:
       - name: setup-graalvm-ce
         uses: rinx/setup-graalvm-ce@v0.0.5
         with:
-          graalvm-version: "20.2.0"
+          graalvm-version: "20.3.0"
           java-version: "java11"
           native-image: "true"
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-m2
       - name: Build and test with Maven
         run: |
-          cd customer-service
-          ./build-native.sh
\ No newline at end of file
+          cd ${{ env.SERVICE_NAME }}
+          ./build.sh
```

Comment:
Warum hier nur build.sh und nicht build-native.sh?

Reply:
Damit wir das native image nicht zwei mal bauen. Alternative könnte man den step komplett rausnehmen da in dem docker build, build-native.sh nochmal aufgerufen wird.